### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by kiler129** [constantcontact/php-sdk#153]
This project wasn't updated for 2 years, issues are still open, it still uses old GuzzleHttp 5.x, etc.

So can someone officially answer following questions?
 - Is the PHP integration officially abandoned?
 - Is ConstantContact has any plans to update this SDK to support current version of GuzzleHttp?
 - Is the php-sdk officially supported on PHP 7?

**Comment by tzkmx** [comment]
Yeah ir looks Pretty much as abandonware and I prefer using my own methods to upload data to the laughable API expecting JSON data (thus, UTF8 encoded) but the email body must be encoded in ISO 8859-1... I mean, really?

**Comment by mpryvkin** [comment]
This is the response I got from Tier II API Support Engineer on 2/4/2019:

> The v2 API is not abandoned; however we are not updating the SDKs on it due to our resources being focused on developing and releasing the v3 API. We currently do not have a release time for the email campaigns or reporting for v3. If you need to send emails through the API then I would suggest to use the v2 for that purpose.
>
> While the SDKs are not being updated they are still functional and should be able to work with a minimum of implementation.

**Comment by bkuhl** [comment]
Here's my testing results on this SDK:

 * `3.0-beta` - I've just submitted a [PR to run tests in PHP 7.x versions](https://github.com/constantcontact/php-sdk/pull/160) and they pass (though it's using 5.* guzzle, not the latest)
 * `2.x` - I tested locally against PHP 7.3 and it passes
 * `1.x` - Has no tests, no one should use this anyways

**Comment by phpfui** [comment]
I have implemented a complete V3 API package at (https://packagist.org/packages/phpfui/constantcontact) Full documentation is at [PHPFUI/ConstantContact](http://phpfui.com/?n=PHPFUI%5CConstantContact)
